core/build.gradle

```diff
@@ -55,7 +55,7 @@ tasks.japicmp {
 configurations.all {
     resolutionStrategy {
         // use lower Jackson version
-        force 'com.fasterxml.jackson.core:jackson-databind:2.8.8'
+        force 'com.fasterxml.jackson.core:jackson-databind:2.10.0'
```

Comment:
we forced it for reason, see https://github.com/testcontainers/testcontainers-java/pull/3602

Why does Jib require a bump?

Reply:
it fails because of this without bumping the jackson version

```
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.databind.ser.std.ToStringSerializerBase
	at java.net.URLClassLoader.findClass(URLClassLoader.java:387)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	... 23 more
```

Reply:
I have forced `com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.8.8`